### PR TITLE
CON 576: Update Piston Docs

### PR DESCRIPTION
**Pull request by kayladnls**
- [x] Removed closure based routing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,24 +8,25 @@ Opinionated Micro Framework for APIs
 
 ## Routing
 
-Piston supports route based and closure based routing. In both cases, the action must return an instance of `Refinery29\Piston\Http\Request`. Routes are implemented as simple value objects that hold url alias, http verb, and action. 
+Piston supports class based routing. The class being routed to must return an instance of `Refinery29\Piston\Http\Request`. Routes are implemented as simple value objects that hold url alias, http verb, and action. 
 
-**Route Based**
 ```php
 $application = new Application();
```
Comment (localheinz) [inline README.md]: @kayladnls 

While we're at it, would you like to change this to `Piston`?

File: README.md [hunk 2]
```diff
@@ -8,24 +8,25 @@ Opinionated Micro Framework for APIs
...
+
+`kayladnls/seesaw` allows for the reverse routing. 
+
+
 ```php
-$application = new Application();
-$application->addRoute(Route::get('jedi/{id}', function($request, $response) {
-	return $response;
-}));
+$application->addNamedRoute('QuiGon', 'GET', 'jedi/quigon', JediController::QuiGon);
```
Comment (localheinz) [inline README.md]: @kayladnls 

Should this be

```php
$application->addNamedRoute('QuiGon', 'GET', 'jedi/quigon', JediController::class);
```

?

**Comment by localheinz** [comment]
@kayladnls 

:+1: Other than the small comment, this looks good to me!

File: README.md (hunk 2 above)
Comment (kayladnls) [inline README.md]: No. `JediController::QuiGon` refers to the function inside the `JediController` that should be called.

**Pull request merged**